inlong-sdk/dataproxy-sdk-twins/dataproxy-sdk-golang/logger/logger.go

```diff
@@ -42,7 +42,7 @@ type stdLogger struct {
 }
 
 func (s stdLogger) Debug(args ...interface{}) {
-	fmt.Println(args...)
+	// fmt.Println(args...)
```

Comment:
Why not remove this line? Or remove this function?

Reply:
Hi @healchow, I keep this line is for the sake of debugging,  when people want to debug, they can uncomment this line

Reply:
Okay~